libcxx/src/support/runtime/exception_pointer_cxxabi.ipp

```diff
@@ -34,6 +35,29 @@ exception_ptr& exception_ptr::operator=(const exception_ptr& other) noexcept
     return *this;
 }
 
+#  if _LIBCPP_AVAILABILITY_HAS_INIT_PRIMARY_EXCEPTION
+void *exception_ptr::__init_native_exception(size_t size, type_info* tinfo, void (_LIBCXX_DTOR_FUNC* dest)(void*)) noexcept
```

Comment:
Separately from the above comment, I actually wonder if we should be defining these methods in the dylib at all. Is there a reason why we're not simply calling `__cxa_allocate_exception` and `__cxa_init_primary_exception` directly from the headers?

Reply:
I thought one of the purposes of `exception_pointer_*.ipp` being in dylib is to not call `__cxa_*` from headers, but if it's not the case then we definitely can call them directly from headers.

Done just that.